### PR TITLE
Add MAX Framework Package

### PR DESCRIPTION
**Pull request by bdwyer2**
~TODO: figure out how to handle passing in multiple variables to `predict()` ([core/model.py#68](https://github.com/IBM/MAX-Fast-Neural-Style-Transfer/pull/21/files#diff-0831a56d210514c03372021d9e8f19fbR68))~

### CODE REVIEW

File: core/model.py [hunk 1]
```diff
@@ -65,7 +65,11 @@ def _post_process(self, x):
     def _load_assets(self, path):
         pass
 
-    def predict(self, x, model):
+    def predict(self, args):
```
Comment (djalova) [inline core/model.py]: I think we should move `_pre_process()` out of this so it can be refactored to `_predict()`.

File: core/model.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            trn.ToTensor(),
+            trn.Lambda(lambda x: x.mul(255))
+        ])
+        image = V(content_transform(image).unsqueeze(0), volatile=True)
+        return {'image': image, 'model': args['model']}
+
+    def _post_process(self, x):
+        return Image.fromarray(x)
+
+    def _load_assets(self, path):
```
Comment (djalova) [inline core/model.py]: Is this used anywhere?

Comment (bdwyer2) [reply core/model.py]: It's used on line 43

Comment (bdwyer2) [reply core/model.py]: Not sure what it's suppose to do

Comment (bdwyer2) [reply core/model.py]: Tests pass without it. I pushed a commit to remove it.

**Pull request merged**